src/playback-watcher.js

```diff
@@ -219,18 +218,6 @@ export default class PlaybackWatcher {
     this.resetSegmentDownloads_(type);
     this.tech_.trigger({type: 'usage', name: `vhs-${type}-download-exclusion`});
 
-    if (type === 'subtitle') {
```

Comment:
I think by removing this we will still try to blacklist subtitle tracks when we don't have subtitles in the segments. Should we just return early for subtitles tracks, or, in loaderTypes above, remove subtitles?

Reply:
oh, we'll still try to exclude it but at least it won't be removed. Yeah, exiting early here when it's of `type` `subtitle` sounds good.

Reply:
Updated to return here but not remove the track.